WORKSPACE

```diff
@@ -52,7 +52,7 @@ _piptool_install()
 git_repository(
     name = "subpar",
     remote = "https://github.com/google/subpar",
-    tag = "1.0.0",
+    commit = "b1b6bd13ff80ede09f57539031cfb20459c28237",
```

Comment:
I tagged 1.1.0 if you want to use that instead (You'll miss the CI changes which are irrelevant here).